tools/release-init.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+realm_version="$1"
```

Comment:
Can you check this input before use? At least check if there was one argument to the program and maybe if it looks something like "x.x.x" and fail and exit otherwise. It would be nice to prevent pushing branches if the shell script is run with no arguments.